docs/security.html

```diff
@@ -733,11 +733,11 @@
     </ol>
     Here is an example of how to run the migration tool:
     <pre>
-    ./bin/zookeeper-security-migration --zookeeper.acl=secure --zookeeper.connection=localhost:2181
```

Comment:
The comments at the very beginning in ZkSecurityMigrator.scala should also be updated.